dom/src/test/java/org/apache/james/mime4j/message/MessageBuilderTest.java

```diff
@@ -46,7 +47,7 @@
 
     @Test
     public void testSetCustomBody() throws Exception {
-        MessageBuilder builder = MessageBuilder.create();
+        MessageBuilder builder = MessageBuilder.of();
```

Comment:
Ok after speaking with you on mattermost we agreed that it would be better to deprecate the all builder and replace it by a new builder which has a totally different type than the old one in order to avoid old code use the new builder!
I worked on your idea and pushed a new commit: https://github.com/linagora/james-mime4j/pull/6/commits/810e3eb3b40330d89f417f778354672243b09b61.
I think this result in a very `ofuscated commit` with a lot of code.. I would prefer to stick to the version before this commit what do you think ?